caliban/hooks/examples/git_hooks.py

```diff
@@ -0,0 +1,76 @@
+"""Pre-build and pre-run hooks, written as python functions
+Each function is a separate hook, that can be enabled
+or disabled by adding or removing the function name from
+the pre_build_hooks or pre_run_hooks entry in the .calibanconfig.json file
+
+Notes:
+  1.  Hook names must begin with 'hook_'
```

Comment:
As mentioned elsewhere do we really want to implement the hook_ convention as a hard constraint?